<!DOCTYPE html>
<html>
  <head>
    <title>W3.CSS Template</title>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    <style>
    body, h1,h2,h3,h4,h5,h6 {font-family: "Montserrat", sans-serif}
    .w3-row-padding img {margin-bottom: 12px}
    /* Set the width of the sidebar to 120px */
    .w3-sidebar {width: 120px;background: #cccccc;}
    /* Add a left margin to the "page content" that matches the width of the sidebar (120px) */
    #main {margin-left: 120px}
    /* Remove margins from "page content" on small screens */
    @media only screen and (max-width: 600px) {#main {margin-left: 0}}
    </style>
  </head>
<body class="w3-grey">

<!-- Icon Bar (Sidebar - hidden on small screens) -->
<nav class="w3-sidebar w3-bar-block w3-small w3-hide-small w3-center">
  <!-- Avatar image in top left corner -->
 <h1 class=<span class="w3-hide-small">X.L.</h1>
  <a href="#" class="w3-bar-item w3-button w3-padding-large w3-grey">
    <i class="fa fa-home w3-xxlarge"></i>
    <p>HOME</p>
  </a>
  <a href="#short CV" class="w3-bar-item w3-button w3-padding-large w3-hover-grey">
    <i class="fa fa-user w3-xxlarge"></i>
    <p>Short CV</p>
  </a>
  <a href="#Research" class="w3-bar-item w3-button w3-padding-large w3-hover-Gainsboro">
    <i class="fa fa-eye w3-xxlarge"></i>
    <p>Research</p>
  </a>
  <a href="#tools" class="w3-bar-item w3-button w3-padding-large w3-hover-grey">
    <i class="fa fa-book w3-xxlarge"></i>
    <p>Tools</p>
  </a>
    <a href="#contact" class="w3-bar-item w3-button w3-padding-large w3-hover-grey">
    <i class="fa fa-envelope w3-xxlarge"></i>
    <p>Contact</p>
  </a>
</nav>

<!-- Navbar on small screens (Hidden on medium and large screens) -->
<div class="w3-top w3-hide-large w3-hide-medium" id="myNavbar">
  <div class="w3-bar w3-grey w3-opacity w3-hover-opacity-off w3-center w3-small">
    <a href="#" class="w3-bar-item w3-button" style="width:25% !important">HOME</a>
    <a href="#short CV" class="w3-bar-item w3-button" style="width:25% !important">Short CV</a>
    <a href="#research" class="w3-bar-item w3-button" style="width:25% !important">Research</a>
    <a href="#tools" class="w3-bar-item w3-button" style="width:25% !important">Tools</a>
    <a href="#contact" class="w3-bar-item w3-button" style="width:25% !important">contact</a>
  </div>
</div>

<!-- Page Content -->
<div class="w3-padding-large" id="main">
  <!-- Header/Home -->
  <header class="w3-container w3-padding-32 w3-center w3-grey" id="home">
    <h1 class="w3-jumbo"><span class="w3-hide-small">I'm</span> Xiaolin Liu.</h1>
    <p>I will work on bacteria to study behavior if only I know how to tame them.  --Max Delbruck</p>
    <img src="
              xiaolin_.jpg
              
              " alt="boy" class="w3-image" width="248" height="277">

    
        <p><a href="mailto:xliu241@ucsc.edu">Email</a>&nbsp/&nbsp
          <a href="https://www.researchgate.net/profile/Xiaolin-Liu-4">ResearchGate</a>&nbsp/&nbsp
    <a href="https://orcid.org/0000-0001-5891-4559">ORCiD</a> </p>
  </header>

  <!-- Short CV Section -->
 
 <div class="w3-padding-64 w3-content"" id="short CV">
    

  <h2 class="w3-text-light-grey">Experience</h2>
  <hr style="width:200px" class="w3-opacity">
  <p class="w3-justify">
    <p><li> 2022.01-present, University of California, Santa Cruz, Postdoc</a>.</li></p>
    <p><li> 2019.10-2021.12, University of California, Santa Cruz, Visiting student</a>.</li></p>
    <p><li> 2015.09-2021.12, Yantai Institute of Coastal Zone Research, CAS, Yantai, Ph. D</a>.</li></p>
    <p><li> 2015.09-2021.12, University of Chinese Academy of Sciences, Beijing, Ph. D</a>.</li></p>
    <p><li> 2011.09-2015.06, Shandong Normal University, Jinan, B.S</a>.</li></p>
    </p>

    <h3 class="w3-padding-16 w3-text-light-grey">Presentation</h3>
    <hr style="width:200px" class="w3-opacity">
       <p class="w3-justify">
      <p><li> 2022.11, Poster Presentation: Chemotaxis system regulated flagellar rotation, but not chemotaxis, contributes to the biofilm formation of Helicobacter pylori. @ASM Biofilm 2022, 2022, Charlotte, NC, US.</li></p>
      <p><li> 2022.09, Poster Presentation: FliL has a conserved function in diverse microbes to negatively modulate motor output via its N-terminal region. @Signal Transduction in Microorganism, Gordon Research Conference (GRC), 2022, Ventura, CA, US.</li></p>
      <p><li> 2022.05, Poster Presentation: Methylation-independent chemotaxis systems are the norm for gastric-colonizing bacteria. @UC Santa Cruz ANNUAL POSTDOC SYMPOSIUM 2022, Santa Cruz, CA, US.</li></p>

      <p><li> 2021.01, Oral Presentation: Chemotaxis plays positive and negative roles on biofilm formation in Helicobacter pylori. @Bacterial Locomotion and Signal Transduction (BLAST), 2021, Virtual, US.</li></p>
      <p><li> 2020.01, Poster Presentation: Boost or Inhibit: Oxalic acid from Sesbania rostrata seed exudates mediates the chemotactic response of Azorhizobium caulinodans ORS571 using multiple strategies. @Signal Transduction in Microorganism, Gordon Research Conference (GRC), 2020, Ventura, CA, US.</li></p>

        </p> 

    <h3 class="w3-padding-16 w3-text-light-grey">Awards</h3>
    <hr style="width:200px" class="w3-opacity">
                  <p class="w3-justify">
        </p>
        <p class="w3-justify">
<p><li> 2020, UCAS-BHBP Scholarship </p>
<p><li> 2019, National Scholarship for Graduate Students </p>
<p><li> 2019-2021, Scholarship from China Scholarship Council </p>
<p><li> 2019, Di-Ao Scholarship </p>
<p><li> 2018, Liu Ruiyu Marine Science Scholarship </p>
<p><li> 2017,2018, Level A Academic Scholarship for Graduate Students </p>

        </p> 

    
    </div>

    

  
  <!-- Resarch Section -->
  
      <div class="w3-padding-64 w3-content"" id="Research">
    <h2 class="w3-text-light-grey">Research</h2> 
     <hr style="width:200px" class="w3-opacity">
   
                 I‘m interested in bacterial motility and chemotaxis, including how do bacteria sense extracellular signals, what ligand can be sensed, identifying novel chemotaxis proteins, and how do flagella sense mechanical signals. I’m also curious about biofilm. Biofilms are the major habitats of sessile bacteria on the surface of the earth. I’m interested in how does bacterium form and regulate biofilm, what’s the function of chemotaxis on the attachment and detachment processes of biofilm, and how does bacterium balance its planktonic (motile/chemotaxis) and sessile (biofilm) lifestyles  
   </p>
   
  
  <h4> Helicobacter pylori chemotaxis and biofilm formation </h4>
   

    <ol>
       <p>
      <li><strong>Methylation-independent chemotaxis systems are the norm for gastric-colonizing Helicobacter species</strong>
      <br><strong>Xiaolin Liu</strong>, Karen M. Ottemann  <br>
      <em>Journal of Bacteriology</em> 2022 <em>(Editor's Pick)</em>| <a style="color: #447ec9" href="https://journals.asm.org/doi/10.1128/jb.00231-22">paper</a>  
      </p>
     
  
    
      <p>
      <li><strong>FliL has a conserved function in diverse microbes to negatively modulate motor output via its N-terminal region</strong>
      <br>
     <strong>Xiaolin Liu</strong>, Anna Roujeinikova, Karen M. Ottemann 
      <br>
      <em>BioRiv</em> 2022 | <a style="color: #447ec9" href="https://www.biorxiv.org/content/10.1101/2022.05.17.492383v1">paper</a> 
      </p>
   

         <p>
      <li><strong>The flagellar motor protein FliL forms a scaffold of circumferentially positioned rings required for stator activation</strong>
      <br>
     Shoichi Tachiyamaa, Kar L. Chan (co-first author), <strong>Xiaolin Liu</strong>, Skander Hathroubi, Briana Peterson, Mohammad F. Khan, Karen M. Ottemann, Jun Liu, Anna Roujeinikova
      <br>
      <em>Proceedings of the National Academy of Sciences of the United States of America</em> 2022 | <a style="color: #447ec9" href="https://www.pnas.org/doi/10.1073/pnas.2118401119">paper</a> 
      </p>
      </ol>

    <h4> Azorhizobium caulinodans chemotaxis and host colonization </h4>
    

    <ol>
     
                         <p>
      <li><strong>Systematic Analysis of Lysine Acetylation Reveals Diverse Functions in Azorhizobium caulinodans Strain ORS571</strong>
      <br>
     Yanan Liu, <strong>Xiaolin Liu</strong>, Xiaoyan Dong, zhiqiu Yin, Zhihong Xie, Yongming Luo
      <br>
      <em>Microbiology Spectrum</em> 2022 | <a style="color: #447ec9" href="https://journals.asm.org/doi/10.1128/spectrum.03539-22">paper</a> 
      </p>
      
                   <p>
      <li><strong>The effect of Azorhizobium caulinodans ORS571 and γ-aminobutyric acid on salt tolerance of Sesbania rostrata</strong>
      <br>
     Yanan Liu, <strong>Xiaolin Liu</strong>, Xiaoyan Dong, Jiaming Yan, Zhihong Xie, Yongming Luo
      <br>
      <em>Frontiers in Microbiology</em> 2022 | <a style="color: #447ec9" href="https://www.frontiersin.org/articles/10.3389/fpls.2022.926850/full">paper</a> 
      </p>
      
      <p>
      <li><strong>Azorhizobium caulinodans chemotaxis is controlled by an unusual phosphorelay network</strong>
      <br>
     Emily N. Kennedy, Sarah A. Barr, <strong>Xiaolin Liu</strong>, Luke R. Vass, Yanan Liu, Zhihong Xie, Robert B. Bourret
      <br>
      <em>Journal of Bacteriology</em> 2022 | <a style="color: #447ec9" href="https://journals.asm.org/doi/10.1128/jb.00527-21">paper</a> 
      </p>

      <p>
      <li><strong>The hypoxia-associated localization of chemotaxis protein CheZ in Azorhizorbium caulinodans</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Yanan Liu, Yixuan Wang, Dandan Wang, Kevin Scot Johnson, Zhihong Xie
      <br>
      <em>Frontiers in Microbiology</em> 2021 | <a style="color: #447ec9" href="https://www.frontiersin.org/articles/10.3389/fmicb.2021.731419/full">paper</a> 
      </p>

   <p>
      <li><strong>Protein residues and a novel motif involved in the cellular localization of CheZ in Azorhizobium caulinodans ORS571</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Yanan Liu, Kevin Scot Johnson, Xiaoyan Dong, Zhihong Xie
      <br>
      <em>Frontiers in Microbiology</em> 2020 | <a style="color: #447ec9" href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.585140/full">paper</a> 
      </p>
      
         
      
       <p>
      <li><strong>Effects of calcium and signal sensing systems on Azorhizobium caulinodans biofilm formation and host colonization</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Kaiye Zhang (co-first author), Yanan Liu, Desheng Zou, Dandan Wang, Zhihong Xie
      <br>
      <em>Frontiers in Microbiology</em> 2020 | <a style="color: #447ec9" href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.563367/full">paper</a> 
      </p>
      
      
          <p>
      <li><strong>Azorhizobium caulinodans c-di-GMP phosphodiesterase Chp1 involved in motility, EPS production, and nodulation of the host plant</strong>
      <br>
Yu Sun, Yanan Liu (co-first author), <strong>Xiaolin Liu</strong>, Xiaoxiao Dang, Xiaoyan Dong, Zhihong Xie
      <br>
      <em>Applied Microbiology and Biotechnology</em> 2020 | <a style="color: #447ec9" href="https://link.springer.com/article/10.1007/s00253-020-10404-6">paper</a> 
      </p>
      
      
          <p>
      <li><strong>Oxalic acid from Sesbania rostrata seed exudates mediates the chemotactic response of Azorhizobium caulinodans ORS571 using multiple strategies</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Kaiye Zhang, Yanan Liu, Zhihong Xie
      <br>
      <em>Frontiers in Microbiology</em> 2019 | <a style="color: #447ec9" href="https://www.frontiersin.org/articles/10.3389/fmicb.2019.02727/full">paper</a> 
      </p>
      
      
             <p>
      <li><strong>A dual role of amino acids from Sesbania rostrata seed exudates in chemotaxis of Azorhizobium caulinodans ORS571</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Zhihong Xie, Yixuan Wang, Yu Sun, Xiaoxiao Dang, Haishuan Sun
      <br>
      <em>Molecular Plant-Microbe Interaction</em> 2019 | <a style="color: #447ec9" href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-03-19-0059-R">paper</a> 
      </p>
      
         <p>
      <li><strong>Inactivation of the phosphatase CheZ alters cell-surface properties of Azorhizobium caulinodans ORS571 and symbiotic association with Sesbania rostrata</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Zhihong Xie
      <br>
      <em>Molecular Plant-Microbe Interaction</em> 2019 | <a style="color: #447ec9" href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-05-19-0143-R?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed">paper</a> 
      </p>
      
               <p>
      <li><strong>The genome of Ensifer alkalisoli YIC4027 provides insights for host specificity and environmental adaptations</strong>
      <br>
Xiaoxiao Dang, Zhihong Xie, Wei Liu, Yu Sun, <strong>Xiaolin Liu</strong> Yongqiang Zhu, Christian Staehelin
      <br>
      <em>BMC Genomics</em> 2019 | <a style="color: #447ec9" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6004-7">paper</a> 
      </p>
      
          <p>
      <li><strong>A cheZ-like gene in Azorhizobium caulinodans is a key gene in the control of chemotaxis and colonization of the host plant</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Wei Liu, Yu Sun, Chunlei Xia, Claudine Elmerich, Zhihong Xie
      <br>
      <em>Applied and Environmental Microbiology</em> 2018 | <a style="color: #447ec9" href="https://journals.asm.org/doi/full/10.1128/AEM.01827-17">paper</a> 
      </p>
      
          <p>
      <li><strong>A Chemotaxis-Like Pathway of Azorhizobium caulinodans controls flagella-driven motility, which regulates biofilm formation, exopolysaccharide biosynthesis, and competitive nodulation</strong>
      <br>
 Wei Liu, Yu Sun, Rimin Shen, Xiaoxiao Dang, <strong>Xiaolin Liu</strong>, Fu Sui, Yan Li, Zhenpeng Zhang, Gladys Alexandre, Claudine Elmerich, Zhihong Xie
      <br>
      <em>Molecular Plant-Microbe Interaction</em> 2018 | <a style="color: #447ec9" href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-12-17-0290-R">paper</a> 
      </p>
      
      
              <p>
      <li><strong>Plant growth promotion properties of bacterial strains isolated from the rhizosphere of the Jerusalem artichoke (Helianthus tuberosus L.) adapted to saline–alkaline soils and their effect on wheat growth</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Xiangyue Li (co-first author), Yan Li, Runzhi Li, and Zhihong Xie
      <br>
      <em>Canadian Journal of Microbiology</em> 2017 | <a style="color: #447ec9" href="https://cdnsciencepub.com/doi/10.1139/cjm-2016-0511">paper</a> 
      </p>
      
                       <p>
      <li><strong>Azorhizobium caulinodans transmembrane chemoreceptor TlpA1 involved in host colonization and nodulation on roots and stems</strong>
      <br>
Wei Liu, Jinbao Yang, Yu Sun, <strong>Xiaolin Liu</strong>, Yan Li, Zhenpeng Zhang, Zhihong Xie
      <br>
      <em>Frontiers in Microbiology</em> 2017 | <a style="color: #447ec9" href="https://www.frontiersin.org/articles/10.3389/fmicb.2017.01327/full">paper</a> 
      </p>
      
                    <p>
      <li><strong>Advances in phosphatase CheZ of bacterial chemotaxis signaling pathway</strong>
      <br>
 <strong>Xiaolin Liu</strong>, Wei Liu, Zhihong Xie
      <br>
      <em>Acta Microbiologica Sinica</em> 2016 | <a style="color: #447ec9" href="https://pubmed.ncbi.nlm.nih.gov/29746056/">paper</a> 
      </p>
      
      
      
      
      
      
  </div>


  <!-- End Research Section -->


  <!-- Tools Section -->
<div class="w3-padding-64 w3-content"" id="tools">
    <h2 class="w3-text-light-grey">Tools</h2> 
     <hr style="width:200px" class="w3-opacity">
    
   <h4> Bioinformatics Portals </h4>
       <ol>
                    <p><li><a href="https://www.ncbi.nlm.nih.gov/">NCBI</a></p>
                    <p><li><a href="https://www.ebi.ac.uk/">EBI</a></p></p>
                    <p><li><a href="https://c3bi.pasteur.fr/hub/tools/">Pasteur-tools</a></p>
                    <p><li><a href="http://abc.cbi.pku.edu.cn/index.php">abc</a></p>
                    <p><li><a href="https://www.ibi.vu.nl/programs/">IBIVU</a></p>
                    <p><li><a href="https://toolkit.tuebingen.mpg.de/">MPI-toolkit</a></p>
                    <p><li><a href="http://molbiol-tools.ca/">Online-tools</a></p> 
                    <p><li><a href="https://nmdc.cn/">National Microbiology Data Center</a></p>                                         
                          </ol>        
                                                                                                                  
    <h4> Microbial genomic resources </h4>
                                                <ol>
              <p><li><a href="http://www.microbesonline.org/">Microbesonline</a></p>
              <p><li><a href="https://gtdb.ecogenomic.org/">GTDB</a></p></p>
              <p><li><a href="https://mistdb.com/">Mistdb</a></p>
              <p><li><a href="http://microbes.ucsc.edu/">UCSC microbial Genomie browser</a></p>
                                                </ol>      
                                                                                                                  
                                                                                                                  
                                                                                                                  
   <h4> Protein</h4>
                                                                                                                 <ol>
                   <p><li><a href="https://www.uniprot.org/">Uniprot</a>, a comprehensive resource for protein sequence and annotation data</p>
                   <p><li><a href="http://www.ncbi.nlm.nih.gov/cdd/">CDD</a>, Conserved domain database</p>
                   <p><li><a href="https://www.ebi.ac.uk/interpro/">Interpro</a>, Database of protein families instead of old Pfam database</p>
                   <p><li><a href="http://annotree.uwaterloo.ca/app/">Annotree</a>,a web tool to facilitate visualization of genome annotations across large phylogenetic trees</p>   
                   <p><li><a href="http://trend.evobionet.com/">TREND</a>,TRee-based Exploration of Neighborhoods and Domains</p>   
                   <p><li><a href="https://www.ebi.ac.uk/Tools/hmmer/">HMMER</a>, fast and sensitive homology searches</p>
                   <p><li><a href="http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index">Phyre2</a>, Protein Homology/analogY Recognition Engine V 2.0</p> 
                   <p><li><a href="http://cdvist.joulinelab.org/">CD-vist</a>, a sequence-based protein domain search tool</p>
                   <p><li><a href="https://swissmodel.expasy.org/">SWISS-model</a>, a fully automated protein structure homology-modelling server</p>
                   <p><li><a href="https://predictprotein.org/">Predict Protein</a>, prediction for secondary structure, solvent accessibility, transmembrane helices, globular regions, coiled-coil regions</p>
                   <p><li><a href="https://www.compbio.dundee.ac.uk/jpred/">Jpred</a>, Protein Secondary Structure Prediction server</p>
                   <p><li><a href="http://string-db.org/">String</a>, Database of protein association networks</p>
                   <p><li><a href="http://aquerium.adebalilab.org/">Aquerium</a>, A domain-based protein family querier</p>
                   <p><li><a href="https://static-content.springer.com/esm/art%3A10.1038%2F35051615/MediaObjects/41586_2001_BF35051615_MOESM1_ESM.txt">H. pylori protein interaction</a>, published data</p>                                                                                          
                                                                                                                    
                          </ol>        
                                                                                                                  
       <h4> Comparative and evolutionary genomics</h4>
                                                                                                                 <ol>
                   <p><li><a href="https://www.ncbi.nlm.nih.gov/research/cog/">COG</a>, Clusters of Orthologous Genes</p>
                   <p><li><a href="https://biocyc.org/">BioCyc</a>, a database collection and website with extensive search and analysis tools</p>                             
                   <p><li><a href="https://www.evolgenius.info/evolview">Evolview</a>, a webserver for visualization, annotation, and management of phylogenetic trees</p>                                    
                   <p><li><a href="https://bacdive.dsmz.de/">BacDive</a>,  the worldwide largest database for standardized bacterial information</p>                                    
                   <p><li><a href="http://plannotate.barricklab.org/#annotate-your-engineered-plasmids">plannotate</a>, plasmid annotation (feature identification)</p>
                   <p><li><a href="https://www.fruitfly.org/seq_tools/promoter.html">BDGP</a>, promotor prediction</p>                                       
                                                                                                                      
                                                                                                                      </ol>                                
                                                            
                                                                                                                       
   <h4> Bioinformatics study </h4>
                                             <ol>             
          
     <p><li> <a href="http://lectures.molgen.mpg.de/online_lectures.html">MPI lectures on bioinformatics</a></p>
     <p><li> <a href="https://mybiosoftware.com/">Bioinformatic software update</a></p>
     <p><li> <a href="https://www.bilibili.com/video/BV13t411G7oh?p=1&vd_source=64ee3867c0652ec2379058d89d4432b1">Peking University lectures on bioinformatics</a></p>
              </ol>  
     <h4> Molecular biology & Mathmatics</h4>
                          <ol>
     <p><li><a href="https://www.graphpad.com/guides/prism/8/statistics/stat_---_principles_of_statistics_-.htm">PRINCIPLES OF STATISTICS</a></p></p>
     <p><li><a href="https://imagingbook.com/books/english-edition-hardcover/">ImageJ</a></p>
     <p><li><a href="https://r-graph-gallery.com/">R Graph gallery</a></p>
     <p><li><a href="https://www.neb.com/tools-and-resources">NEB resources &tools</a></p>
     <p><li><a href="https://papers.genomics.lbl.gov/cgi-bin/litSearch.cgi">PaperBlast</a></p></p>
     <p><li><a href="https://blog.addgene.org/">Addgene blog</a></p>
                                                                                                                                    
                          </ol>        
                                                                                               
  <!-- End Tools Section -->
  </div>
   

  <!-- Contact Section -->
  

 <div class="w3-padding-64 w3-content"" id="contact">
    <h2 class="w3-text-light-grey">Contact</h2> 
     <hr style="width:200px" class="w3-opacity">



    <div class="w3-section">
      <p><i class="fa fa-map-marker fa-fw w3-text-white w3-xxlarge w3-margin-right"></i> California, US</p>
      <p><i class="fa fa-phone fa-fw w3-text-white w3-xxlarge w3-margin-right"></i> Phone: +1 8312952165</p>
      <p><i class="fa fa-envelope fa-fw w3-text-white w3-xxlarge w3-margin-right"> </i> Email: xliu241@ucsc.edu</p>
    </div><br>
    


  
  <!-- End Contact Section -->
  </div>

<!-- Footer. This section contains an ad for W3Schools Spaces. You can leave it to support us. -->
<footer class="w3-content w3-padding-64 w3-text-grey w3-xlarge">
  

  
  
 <p class="w3-small w3-text-white">
   This page has been accessed at least
<a href="https://xliu241.github.io/#"><img src="http://stuff.mit.edu/cgi/counter/xliu241" alt="several"></a>
times since Aug 31, 2022. 
 <!-- End footer -->
</footer>

<!-- END PAGE CONTENT -->
</div>

</body>
</html>

